sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_aggregates_v1/query.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+  submission_date,
+  service,
+  country,
+  language,
+  app_version,
+  os_name,
+  os_version,
+  COUNT(DISTINCT user_id) AS service_users,
```

Comment:
Worth double-checking whether this pre-aggregation logic holds. If the same user uses multiple services per day for eg. This might double count them in any downstream aggregation.

Reply:
In this particular case does it matter? We want to see per service activity? If the same user used two different services still we'd want to count the activity for both services? But I see your point, I will confirm and adjust accordingly.

Reply:
Ok, I could see that in the Look provided with the request we also did count mobile + desktop users. I added an additional service for `desktop_and_mobile`. Would that work @ANich ?

Reply:
If this table is only meant to be for the linked Look then what was previously here was probably fine as long as it matched the Look query. 

I meant that we want to ensure that the same answer gets returned regardless of whether someone queries this table or the upstream. If the Looker measure is going to end up as a `SUM` then summing distinct users across multiple dimensions doesn't usually work if they're not mutually exclusive. `service` just looked like something that isn't mutually exclusive across a day for eg.

Reply:
@ANich I guess we could add a disclaimer in the table description saying that each category should not be summed together as it will result in overcounting. Do you think that would be enough to proceed?

Reply:
added a disclaimer @ANich, I hope that's enough.

Reply:
Yeah I think a disclaimer could work if you're able to reflect that in Looker as well